src/Service.Tests/Unittests/SerializationDeserializationTests.cs

```diff
@@ -412,7 +412,7 @@ private static void VerifyColumnDefinitionSerializationDeserialization(ColumnDef
         {
             // test number of properties/fields defined in Column Definition
             int fields = typeof(ColumnDefinition).GetFields(BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Instance).Length;
-            Assert.AreEqual(fields, 7);
+            Assert.AreEqual(fields, 8);
```

Comment:
magic number. not sure why this value increased from 7 to 8. I understand that this was existing test, but none of the comments surrounding this code explain the number.

Reply:
new field in `ColumnDefinition` : `public SqlDbType? SqlDbType { get; set; }`